### ISSUE TITLE
Expose `allowed_pending_messages` option in CW Agent config.

### ISSUE THREAD

**Issue by pimlock** [aws/amazon-cloudwatch-agent#109]
Hi there,

Would it be possible to expose `allowed_pending_messages` configuration option in the CW Agent configuration (here: https://github.com/aws/amazon-cloudwatch-agent/blob/758afc8b3d3e5f223e52ee12b132c40d9c7eaeb7/translator/config/schema.go#L338-L358).

We are running into this issue:

```
2020-10-21T19:46:50Z E! Error: statsd message queue full. We have dropped XXX messages so far. You may want to increase allowed_pending_messages in the config
```

And would like to increase that value.

According to this [README](https://github.com/aws/amazon-cloudwatch-agent/blob/e770cd8fbdded909034a06550e8d2e40ebbf43fb/plugins/inputs/statsd/README.md) it should be possible to modify that value through the config file, but I'm not sure if that is available in the CW Agent as well (looks like that README is from telegraf).

Thanks!

**Comment by haojhcwa** [comment]
Yeah, the README is inherited from telegraf, we are currently considering exposing this configuration through the public json config file, we will keep you updated here.

**Comment by pimlock** [comment]
Thank you for resolving it so quickly!

How often are new releases published? When can we expect this to be available?

**Comment by haojhcwa** [comment]
The release process usually take about 2 to 3 week for various tests we have, and I believe we are going to start a new release fairly soon.
